### PR TITLE
fix(mediator): Update the readme and docker with type of key forma…

### PR DESCRIPTION
**Pull request by mineme0110**
…t used for mediator identity

### Description: 
Summarize the changes you're submitting in a few sentences, including Jira ticket ATL-xxxx if applicable.
Link to any discussion, related issues and bug reports to give the context to help the reviewer understand the PR.

### Alternatives Considered (optional): 
Link to existing ADR (Architecture Decision Record), if any. If relevant, describe other approaches explored and the selected approach. Documenting why the methods were not selected will create a knowledge base for future reference, helping prevent others from revisiting less optimal ideas.

### Checklist: 
- [X] My PR follows the contribution guidelines of this project
- [X] My PR is free of third-party dependencies that don't comply with the [Allowlist](https://toc.hyperledger.org/governing-documents/allowed-third-party-license-policy.html#approved-licenses-for-allowlist)
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] I have added tests that prove my fix is effective or that my feature works
- [] I have checked the PR title to follow the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/)

### CODE REVIEW

**Review by petevielhaber — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ So in order to configure the mediator for your needs.
 You can either change the default configuration or you can set up environment variables that overrides the defaults:
 
 #### identity
+# KEY_AGREEMENT KEY_AUTHENTICATION are using format JOSE(JWK) OKP type base64urlsafe encoded keys https://www.rfc-editor.org/rfc/rfc8037.html
```
Comment (FabioPinheiro) [inline README.md]: This will be a title in markdown. I'm assuming that is not what you want.

**Review by FabioPinheiro — APPROVED** [review]

**Comment by FabioPinheiro** [comment]
What do you think about also removing the default keys for #264 ?
And switch for a tutorial how to create the keys (2 or 3 paragraphs maximum)

**Comment by mineme0110** [comment]
> What do you think about also removing the default keys for #264 ? And switch for a tutorial how to create the keys (2 or 3 paragraphs maximum)

I think we should have 2 modes dev and prod as you mentioned, I think for getting started quick guide default keys are better 
And we could a tutorial for genrating keys for PROD purpose

File: build.sbt [hunk 2]
```diff
@@ -241,6 +254,8 @@ lazy val mediator = project
     Assets / WebKeys.packagePrefix := "public/",
     Runtime / managedClasspath += (Assets / packageBin).value,
   )
+  .settings(run / fork := true)
```
Comment (FabioPinheiro) [inline build.sbt:257]: Did you test if this works well with `reStart` sbt command?

**Review by FabioPinheiro — APPROVED** [review]
LGTM

File: build.sbt (hunk 2 above)
Comment (mineme0110) [reply build.sbt:257]: No Good Catch It wont work

**Pull request merged**